### PR TITLE
Fix bug for parent tables with two similar relation names

### PR DESCRIPTION
**Pull request by JKoetsier**

### CODE REVIEW

File: src/schematools/events/full.py [hunk 1]
```diff
@@ -61,7 +62,8 @@ def parent_id_value(self, prepared_row: dict):
 
     @property
     def parent_fields(self):
-        return [f.name for f in self.parent_table.columns if f.name.startswith(self.relation_name)]
+        pattern = re.compile(f"{self.relation_name}_[a-zA-Z]+")
```
Comment (lbam) [inline src/schematools/events/full.py:65]: None of my business, but...

```suggestion
        pattern = re.compile(f"{re.escape(self.relation_name)}_[a-zA-Z]+")
```

Comment (JKoetsier) [reply src/schematools/events/full.py:65]: Didn't know about that method :)

Not needed in this case though, as self.relation_name always consists of letters a-z and possible one or more underscores.

Comment (lbam) [reply src/schematools/events/full.py:65]: ... until you hit a bug elsewhere and it doesn't.

**Review by jjmurre — APPROVED** [review]
LGTM

**Pull request merged**